proposals/xxxx-continous-integration.md

```diff
@@ -0,0 +1,54 @@
+# Configuration management
+
+* Proposal: [SDL-xxxx](xxxx-Configuration-Management.md)
+* Author: [Vadym Kushnirenko](https://github.com/vkushnirenko-luxoft)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core/??]
+
+## Introduction
+
+This proposal is about configuration management system for continuous building and testing code.
+
+## Motivation
+
+The development process should be open and transparent. Any person could deploy it on his own side, use it and propose recomendations about improvements.
+Configuration must be saved as a code, be flexible and be renewal in any moment from public resources.
```

Comment:
@vkushnirenko-luxoft 
- What means "_Configuration must be saved as a code_" ? Does that mean "saved into the file"?
- Perhaps ".. be restored.." or ".. be applied.." instead of "_be renewal_"